sections/editing.include

```diff
@@ -1667,14 +1668,24 @@
   If specified, the value must be a single printable character:
   typically a string one Unicode code point in length.
   
-  <p class="warning">Character references such as <code>accesskey="&#65"</code> are not valid values, 
-  because they will cause no shortcut to be assigned in many browsers. 
-  However certain printable characters that may represent more than one Unicode code point, such as 
-  <code>accesskey="श्र"</code> are valid.</p>
+  <div class="warning">
+    Composed characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form D</a>,
+    such as <code>accesskey="&amp;#x006E;&amp;#x0303;"</code>
+    to assign <samp>ñ</samp> as a shortcut, are not valid values,
+    and in many browsers no shortcut will be assigned. Characters in
+    <a href="https://www.w3.org/International/questions/qa-html-css-normalization">Normalised Form C</a>
+    are valid values. Printable characters that may represent more than one Unicode code point, such as 
+    <code>accesskey="श्र"</code>, are valid values.
   
```

Comment:
Did you try typing two latin characters that don't normalise, such as 'nn'? I can understand why an implementer might automagically normalise shortcuts to NFC as  a safer way of matching keyboard results, since most keyboard input is likely to be precomposed characters. However, you'd expect that the results of a keypress would also be automatically normalised too, at run time, and before matching with available shortcuts, in order to ensure best results for matching.